syne_tune/optimizer/schedulers/searchers/bore/bore.py

```diff
@@ -258,9 +258,9 @@ def _train_model(self, train_data, train_targets):
             self.model = CalibratedClassifierCV(
                 self.model, cv=2, method=self.calibration
             )
-            self.model.fit(X, np.array(z, dtype=np.int))
+            self.model.fit(X, np.array(z, dtype=np.int_))
```

Comment:
is this the recommend way by Numpy now? It looks a bit odd with the underscore

Reply:
Please use the other PR I opened. We want to have np.int64 here.

Reply:
yesh approved this. I think int_ is the updated version of np.int - I agree int64 is probably safer but didnt want to change the semantics of the code.